src/entities/transactions/GasFeeType.ts

```diff
@@ -0,0 +1,9 @@
+export enum GasFeeType {
+  current = 2,
```

Comment:
this looks like clone of eip1559TransactionType - actually can we use this instead, the name is nicer

and just be aware of using GasFeeType vs GasFeeTypes depending on TS vs JS files
and remember to export in transactions/index as well as entities/index

Reply:
yeah i changed the name, will roll it back